lib/Resque/Job.php

```diff
@@ -288,14 +294,26 @@ public function setJobFactory(Resque_Job_FactoryInterface $jobFactory)
 		return $this;
 	}
 
-    /**
-     * @return Resque_Job_FactoryInterface
-     */
-    public function getJobFactory()
-    {
-        if ($this->jobFactory === null) {
-            $this->jobFactory = new Resque_Job_Factory();
-        }
-        return $this->jobFactory;
-    }
+	/**
+	 * @return Resque_Job_FactoryInterface
+	 */
+	public function getJobFactory()
+	{
+		if ($this->jobFactory === null) {
+			$this->jobFactory = new Resque_Job_Factory();
+		}
+		return $this->jobFactory;
+	}
```

Comment:
The file is using tabs but these few lines were indented with spaces, so corrected that.